src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -183,6 +184,16 @@ - (instancetype)initWithRequestOptions:(GRPCRequestOptions *)requestOptions
     if (globalInterceptorFactory != nil) {
       [interceptorFactories addObject:globalInterceptorFactory];
     }
+    if (_actualCallOptions.transport != NULL) {
+      id<GRPCTransportFactory> transportFactory = [[GRPCTransportRegistry sharedInstance]
+          getTransportFactoryWithID:_actualCallOptions.transport];
```

Comment:
Not related to this PR, but I was wondering why we use `_callOptions-` instead of `_actualCallOptions` in L200 and L207 below.,